doc/conseil.sql

```diff
@@ -80,7 +80,8 @@ CREATE TABLE public.balance_updates (
     change numeric NOT NULL,
     level numeric,
     delegate character varying,
-    category character varying
+    category character varying,
+    operation_group_hash character varying
```

Comment:
Is this actually needed? see [this note](https://github.com/Cryptonomic/Conseil/issues/471#issuecomment-525271046)

Reply:
So the issue is I wanted a quick fix, and as far as I saw in the table in dev, source_hash only works for sources that are blocks. When I tried to do a similar approach to place operation hashes in source_hash, I ended up somehow not capturing block sources, so I chose the easier way out this time.

Reply:
I'm not sure if I'm missing anything, but I was trying to point out that, for balance updates referring to "operation" or "operation_result" in the `balance_updates.source` column, the referred operation can be found by reading the `operation_id` written in the `balance_updates.source_id` column.

I see no need of the operation group hash, you can simply get the operations row and read the operation group hash from that...

Reply:
True, but we're leaning towards a more normalized data model with as few joins as possible, iirc. @anonymoussprocket thoughts?

Reply:
I'm pretty sure this has to do with ease of use for Arronax specifically. Correct me if I'm wrong.